manifests/g/Google/Chrome/Canary/124.0.6323.0/Google.Chrome.Canary.installer.yaml

```diff
@@ -0,0 +1,65 @@
+# Created with YamlCreate.ps1 v2.3.4 Dumplings Mod $debug=QUSU.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Google.Chrome.Canary
+PackageVersion: 124.0.6323.0
+InstallerType: exe
+InstallModes:
+- silent
+InstallerSwitches:
+  Log: --verbose-logging --log-file="<LOGPATH>"
+ExpectedReturnCodes:
+- InstallerReturnCode: 3
+  ReturnResponse: alreadyInstalled
+- InstallerReturnCode: 4
+  ReturnResponse: downgrade
+- InstallerReturnCode: 22
+  ReturnResponse: cancelledByUser
+- InstallerReturnCode: 60
+  ReturnResponse: installInProgress
+UpgradeBehavior: install
+Protocols:
+- http
+- https
+- mailto
+- tel
+FileExtensions:
+- htm
+- html
+- pdf
+- shtml
+- svg
+- webp
+- xht
+- xhtml
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://dl.google.com/release2/chrome/adarmluiggvp272qjxihafqeztcq_124.0.6323.0/124.0.6323.0_chrome_installer.exe
+  InstallerSha256: C420D5B0AB1BC5FBAF74A2F691F8495C4E1466919DAC61E1B6C5DC053B35E874
+  InstallerSwitches:
+    Custom: --chrome-sxs --do-not-launch-chrome
+  ProductCode: Google Chrome SxS
+# - Architecture: x86
+#   Scope: machine
+#   InstallerUrl: https://dl.google.com/release2/chrome/adarmluiggvp272qjxihafqeztcq_124.0.6323.0/124.0.6323.0_chrome_installer.exe
+#   InstallerSha256: C420D5B0AB1BC5FBAF74A2F691F8495C4E1466919DAC61E1B6C5DC053B35E874
+#   InstallerSwitches:
+#     Custom: --chrome-sxs --do-not-launch-chrome --system-level
+#   ProductCode: Google Chrome SxS

```

Comment:
```suggestion
 - Architecture: x86
   Scope: machine
   InstallerUrl: https://dl.google.com/release2/chrome/adarmluiggvp272qjxihafqeztcq_124.0.6323.0/124.0.6323.0_chrome_installer.exe
   InstallerSha256: C420D5B0AB1BC5FBAF74A2F691F8495C4E1466919DAC61E1B6C5DC053B35E874
   InstallerSwitches:
     Custom: --chrome-sxs --do-not-launch-chrome --system-level
   ProductCode: Google Chrome SxS
```

Reply:
@stephengille There are some issues with the `--system-level` switch which blocked installation and caused previous validations failed.

Reply:
Turns out that Chrome Canary doesn't support installing in machine scope by using the `--system-level` switch. The entries have been removed now.